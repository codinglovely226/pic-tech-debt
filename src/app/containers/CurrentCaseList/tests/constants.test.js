import {
  UPDATE_CASE_LIST_SEARCH_CALLER_LOCATION,
  UPDATE_CASE_LIST_SEARCH_CALLER_ORGANISATION,
  UPDATE_CASE_LIST_SEARCH_CENTRE_NAME,
  UPDATE_CASE_LIST_SEARCH_FILTER,
  UPDATE_CASE_LIST_SEARCH_ID,
  UPDATE_CASE_LIST_SEARCH_NAME,
  UPDATE_CASE_LIST_SEARCH_POSTCODE,
  UPDATE_CASE_LIST_SEARCH_SEX,
  UPDATE_CASE_LIST_SEARCH_TIME,
} from '../constants';

describe('Patient action constants', () => {
  it('UPDATE_CASE_LIST_SEARCH_CALLER_LOCATION', () => {
    expect(UPDATE_CASE_LIST_SEARCH_CALLER_LOCATION).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_CALLER_ORGANISATION', () => {
    expect(UPDATE_CASE_LIST_SEARCH_CALLER_ORGANISATION).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_CENTRE_NAME', () => {
    expect(UPDATE_CASE_LIST_SEARCH_CENTRE_NAME).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_FILTER', () => {
    expect(UPDATE_CASE_LIST_SEARCH_FILTER).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_ID', () => {
    expect(UPDATE_CASE_LIST_SEARCH_ID).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_NAME', () => {
    expect(UPDATE_CASE_LIST_SEARCH_NAME).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_POSTCODE', () => {
    expect(UPDATE_CASE_LIST_SEARCH_POSTCODE).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_SEX', () => {
    expect(UPDATE_CASE_LIST_SEARCH_SEX).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_TIME', () => {
    expect(UPDATE_CASE_LIST_SEARCH_TIME).toMatchSnapshot();
  });
});
