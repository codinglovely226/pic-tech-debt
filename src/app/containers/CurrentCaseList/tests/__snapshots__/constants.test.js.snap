exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_CALLER_LOCATION 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_CALLER_LOCATION"`;

exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_CALLER_ORGANISATION 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_CALLER_ORGANISATION"`;

exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_CENTRE_NAME 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_CENTRE_NAME"`;

exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_FILTER 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_FILTER"`;

exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_ID 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_ID"`;

exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_NAME 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_NAME"`;

exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_POSTCODE 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_POSTCODE"`;

exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_SEX 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_SEX"`;

exports[`Patient action constants UPDATE_CASE_LIST_SEARCH_TIME 1`] = `"pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_TIME"`;
